addons/binding/org.openhab.binding.tankerkoenig/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you have a properly set up OH2 IDE, it should be automatically applied upon save of a file.
Otherwise, try a Ctrl+Shift+F (or context menu->Source->Format).